packages/knip/src/reporters/githubActions.ts

```diff
@@ -0,0 +1,32 @@
+import core from '@actions/core';
+import { ISSUE_TYPES, ISSUE_TYPE_TITLE } from 'src/constants.js';
```

Comment:
relative imports :)

Reply:
As in my original draft, you cannot use ISSUE_TYPES directly because it has `files` and not `_files`.

Reply:
I see that the `disclosure` reporter works around this like so:

https://github.com/webpro-nl/knip/blob/5c91dfa1ce53c4edae4177011e14f4f20eb2da79/packages/knip/src/reporters/disclosure.ts#L32

Should I just copy that pattern?

Reply:
i think so